packages/sdk-core/src/modules/database/BacktraceDatabaseFileRecord.ts

```diff
@@ -24,19 +26,25 @@ export class BacktraceDatabaseFileRecord implements BacktraceDatabaseRecord {
     public static fromRecord(record: BacktraceDatabaseRecord) {
         return new BacktraceDatabaseFileRecord(
             record,
-            record.attachments.filter((n) => n instanceof BacktraceFileAttachment) as BacktraceFileAttachment[],
+            record.attachments.filter(BacktraceDatabaseFileRecord.isFileAttachment),
```

Comment:
Isn't it cleaner we expect to pass File Attachment here instead of doing this inside the class? Now if someone will use it, it would be surprising for someone attachment are not there or are filtered out by the class implementation. WDYT?

Reply:
not sure what you mean by this. I don't have `BacktraceFileAttachment` implementation here. But every implementatio of the interface will return `true` from this filter.